install.sh

```diff
@@ -5,13 +5,23 @@
 
 set -e
 
+for arg in "$@"; do
+    shift
+    case "$arg" in
+        "--root") set -- "$@" "-r" ;;
+        *) set -- "$@" "$arg"
+    esac
+done
```

Comment:
So all we're doing is to change the --root flag into -r?

Reply:
yeah